sky/skylet/job_lib.py

```diff
@@ -86,7 +96,8 @@ def create_table(cursor, conn):
 
     db_utils.add_column_to_table(cursor, conn, 'jobs', 'end_at', 'FLOAT')
     db_utils.add_column_to_table(cursor, conn, 'jobs', 'resources', 'TEXT')
-
+    db_utils.add_column_to_table(cursor, conn, 'jobs', 'pid',
+                                 'INTEGER DEFAULT -1')
```

Comment:
Can we add some comments/docs about what special values PID can take? Something like 
```
0 = not started
-1 = legacy job (using ray)
>0 = running process pid
```
I think it's documented at L588 in update_job_status, but we should probably also add it somewhere close to where the schema is declared.